src/App.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import './App.css';
-import Calculater from './Components/Calculater';
+import { Route, Switch } from 'react-router-dom';
+import Calculator from './pages/Calculator';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
+import Navbar from './pages/Navbar';
 
 class App extends React.PureComponent {
   render() {
     return (
       <div className="App">
-        <h1>
-          Hello from App
-        </h1>
-        <Calculater />
+        <Navbar />
+        <Switch>
+          <Route exact path="/">
 
+            <Home />
+
+          </Route>
+          <Route exact path="/calculator">
+            <Calculator />
```

Comment:
- [OPTIONAL] The text(`Let's do some math!`) should be aligned with the page title(`Math Magicians`) and the caculator should occupy about 2/3 of the page, please kindly fix that. As a front-end dev, it is a very important that you stick to the design as much as possible.